kubernetes/windows-exporter-daemonset.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: win-node-exporter
+  name: win-node-exporter
+  namespace: monitoring
+spec:
+  selector:
+    matchLabels:
+      app: win-node-exporter
+  template:
+    metadata:
+      labels:
+        app: win-node-exporter
+    spec:
+      containers:
+      - args: 
+        - --collectors.enabled=os,iis,container
+        - --telemetry.addr=127.0.0.1:9100
+        name: win-node-exporter
+        image: chzbrgr71/windows_exporter:v1.1
+        ports:
+        - containerPort: 9100
+          hostPort: 9100
+          name: https            
```

Comment:
Does windows_exporter support TLS?

Reply:
I do think TLS and auth need to be handled for this. I saw that TLS came up in the past for this project: https://github.com/prometheus-community/windows_exporter/issues/57 

Also note that the linux prom node_exporter still has TLS as experimental. https://github.com/prometheus/node_exporter#tls-endpoint

Reply:
Cool. I just was confused a bit when tested this solution.